### DISCUSSION TITLE
fetch_mnist.py missing from extra.datasets

### DISCUSSION THREAD

**Discussion by jyothish-mohan** [tinygrad/tinygrad#3822] [Q&A]
I was going through the examples/beautiful_mnist.py script and to get the data it is calling fetch_mnist from extra/datasets but it is missing in extra/datasets

**Comment by borromeotlhs** [comment]
From line 6 in __init__.py in extra/datasets -> def fetch_mnist(tensors=False):

**Reply by jyothish-mohan** [reply]
Thankx

**Comment by chenyuxyz** [comment]
add `PYTHONPATH=.`
